### ISSUE TITLE
I can't achieve the result in your paper. how to achieve the results in your paper, the specific configuration is as follows

### ISSUE THREAD

**Issue by wcq19941215** [YapengTian/AVE-ECCV18#18]
I used the visual_feature.h5 and audio_feature.h5 that you provided. The test result under AV_att is 61.5, and 72.7 in your paper

I use nb_epoch = 500
pytorch version is 1.0.1
The operating system version is  Ubuntu 18.04.4 LTS (GNU/Linux 5.3.0-61-generic x86_64)
Tensorflow version is 1.15.1
cuda version is 10.0

Because the pytorch 0.3.0 you provided is too old, my computer does not support it.
I hope you can help me solve the problem that the A+V-att 72.7 provided in your paper cannot be reproduced
Thank you

**Comment by YapengTian** [comment]
I think you might change the code accordingly if you use pytorch1.0.1. The code was released 2 years ago, so I used 0.3 and released the code. Any issues when you run it in 1.O?

**Comment by wcq19941215** [comment]
There is no anny issues when i use pytorch 1.0.1



---Original---

**Comment by YapengTian** [comment]
Really! I tried the code training with pytorch1.0. But I got an error due to the version issue. I have figured it out and am tuning the model with pytorch1.0 (seems that we need to use different parameters due to version changing).

**Comment by wcq19941215** [comment]
Okay, thank you very much for your experiment. If you have any follow-up improvements, I hope you will let me know. This is my email: 1366901050@qq.com. I hope to have further communication with you.

**Comment by YapengTian** [comment]
Ok. The code will have runtimeerror when using 1.0. After I fixed it, I simply got one model with 71.5% accuracy. It is pretty weird that you only obtained 61.5%.
![image](https://user-images.githubusercontent.com/12854360/87389676-059c1680-c55c-11ea-9719-c4b6c2ceaae4.png)

**Comment by wcq19941215** [comment]
I used the features provided in github, and the test result was 0.45. Then I used the provided method to extract the video features, and then used ffmpeg to extract the audio from mp4 (ffmpeg -y -i xxx.mp4- ar 16000 -ac 1 xxx.wav) and then use the latest vggish provided by tf to extract audio features, so the accuracy is 0.61

**Comment by YapengTian** [comment]
You mean the testing result from the given weights was 0.45? It has ~72.8% with pytorch031. The features are good if you do not want to change feature extractors.

**Comment by wcq19941215** [comment]
I use the weights and features provided in your github, and the video data, the final result is 0.45

**Comment by YapengTian** [comment]
This is really weird. The code has been used by many people. This is the first time I learn this.

**Comment by wcq19941215** [comment]
could you tell me the version of keras?&nbsp;&nbsp;
If you can provide pytorch 1.0 version of the corresponding video and audio features, or tell me the latest extraction method, although I can extract, but may not keep with you, so the effect is not as good as you, thank you

---Original---

**Comment by YapengTian** [comment]
I used Keras-2.0. For implementation with Pytorch 1.0, I use the same features shared here (I never change features). Now, I obtain a model that can achieve 72.5%.
![image](https://user-images.githubusercontent.com/12854360/87731917-73288c80-c780-11ea-9696-86248a358456.png)

**Comment by YapengTian** [comment]
However, it seems that the model trained with 1.0 is more unstable. Different runnings achieve pretty different results. My py1.0 code and pre-trained model can be accessed by this link: https://drive.google.com/file/d/12HW-VPRWCCb3ILZVJycNMU-7YvvzNZXU/view?usp=sharing

**Comment by wcq19941215** [comment]
![image](https://user-images.githubusercontent.com/35086262/87844465-ec92bd00-c8ef-11ea-8ddb-92dca50ea3ed.png)
This is the result I got after training with the code you provided

![image](https://user-images.githubusercontent.com/35086262/87844500-5b701600-c8f0-11ea-9ba0-7de8507ef132.png)
I used the training model you provided and this problem occurred

**Comment by YapengTian** [comment]
I did not get this Error before. And the code works well under pytorch 1.2.0 in my Ubuntu server.  I am not sure whether you followed my instructions to obtain the features for training.

**Comment by wcq19941215** [comment]
Thank you for your patient response. My data is indeed downloaded from your github. The only difference may be the version of some software. The following is the information about the software I am using now：
python                    3.6.10                              
pytorch                   1.2.0     
tensorflow-gpu            1.13.1 
torchvision               0.2.0 
Keras                     2.0.9

**Comment by wcq19941215** [comment]
> I did not get this Error before. And the code works well under pytorch 1.2.0 in my Ubuntu server. I am not sure whether you followed my instructions to obtain the features for training.

When I updated my pytorch version to 1.2.0, the problem was solved, but the test result was only 0.52

**Comment by wcq19941215** [comment]
When I updated my pytorch version to 1.2.0 and retrained for 500 epochs, the test result was 0.69, maybe it’s really just a version problem, thank you for your help

**Comment by YapengTian** [comment]
Great! I am happy to hear that you obtain much better results than before. But it seems that your testing results with my trained model are still bad. I am not sure whether it has certain data precision issue cross different GPUs. I used 1080TI GPU. I had an experience that 2080TI GPU cannot obtain similar results as 1080TI.
